content/en/real_user_monitoring/browser/setup/server/apache.md

```diff
@@ -47,6 +44,8 @@ To automatically instrument your RUM application:
 
 {{% collapse-content title="Manual configuration" level="h5" %}}
 
+If you want to manually load the module onto your web server instead of running the installation script, you can follow the below instructions.
```

Comment:
Nit for conciseness:
```suggestion
To manually load the module onto your web server instead of running the installation script, follow the instructions below.
```